storage/pipeline/commit_batch.go

```diff
@@ -484,6 +675,10 @@ func (b *CommitBatcher) processIndividually(cfg sealiface.Config) ([]sealiface.C
 }
 
 func (b *CommitBatcher) processSingle(cfg sealiface.Config, mi api.MinerInfo, avail *abi.TokenAmount, sn abi.SectorNumber, info AggregateInput, tsk types.TipSetKey) (cid.Cid, error) {
+	return b.processSingleV1(cfg, mi, avail, sn, info, tsk)
+}
+
+func (b *CommitBatcher) processSingleV1(cfg sealiface.Config, mi api.MinerInfo, avail *abi.TokenAmount, sn abi.SectorNumber, info AggregateInput, tsk types.TipSetKey) (cid.Cid, error) {
```

Comment:
@magik6k as mentioned last Friday - I believe it is better for us to use ProveCommitSectcors2 by default. This can help us simplify the  pipeline, and we dont have to go back to  update the pipeline when `provecommitSector` is deprecated (which protocol devs are intended to do to resolve the imbalanced sector activation cost across different methods caused by unwanted network subsidize from cron)

Reply:
Do we know the difference in gas costs between the two?

I want to stay on the conservative side, and use the "old way" until we have proven to ourselves that the new way works and isn't much worse.

If we switch to the new thing by default, we risk breaking SP operations on mainnet, which would not be good.